src/views/404.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+	<div class="flex flex-col items-center bg-gray-100 pt-20">
+		<span class="not-found text-bold text-primary leading-none">
+			404
+		</span>
+		<span class="text-bold text-2xl">
+			Ups! Nič sme nenašli...
+		</span>
+		<router-link
+			to="/"
+			class="bg-primary hover:bg-accent font-bold text-white rounded mt-5 py-2 px-4 cursor-pointer"
+		>
+			Ísť na domov
```

Comment:
"Ísť na domov" by som zmenil asi len na "domov" to ísť mi tam dáko vadí ale inak super, nie je čo vytknúť

Reply:
Nitpick 😄 .

Also, 🇬🇧  please.

Reply:
Yeah it's very subjective